emanifest-py/src/emanifest/client.py

```diff
@@ -855,6 +855,34 @@ def revert_manifest(self, mtn: str) -> RcrainfoResponse:
         endpoint = f"{self.base_url}v1/emanifest/manifest/revert/{mtn}"
         return self.__rcra_request("GET", endpoint)
 
+    def patch_update_manifest(self, mtn: str, body: dict) -> RcrainfoResponse:
+        """
+        Update a portion of a manifest via the patch process
+
+        Args:
+            mtn (str): Manifest tracking number
+            body (dict): Dictionary containing manifest portion details
+
+        Returns:
+            dict: message of success or failure
+        """
+        endpoint = f"{self.base_url}v1/emanifest/manifest/patch-update/{mtn}"
+        return self.__rcra_request("PATCH", endpoint)
+
+    def patch_correct_manifest(self, mtn: str, body: dict) -> RcrainfoResponse:
+        """
+        Update a portion of a manifest via the patch process
+
+        Args:
+            mtn (str): Manifest tracking number
+            body (dict): Dictionary containing manifest portion details
+
+        Returns:
+            dict: message of success or failure
+        """
+        endpoint = f"{self.base_url}v1/emanifest/manifest/patch-correct/{mtn}"
```

Comment:
@wiljnichepa  @scottdchristian are we using the verb ("patch") in the URL for this service? it's kinda an anti-pattern for RESTful designs isn't it? Since that info is already captured in the HTTP verb.

Reply:
@dpgraham4401 @scottdchristian that's the endpoint in sandbox - I will defer to Scott on the final URL